scripts/analyze.sh

```diff
@@ -304,7 +304,23 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
     >&2 echo Using module: Sensor
     MODULES+=" -DBUILD_MODULE_SENSOR"
     echo "export ZJS_SENSOR=y" >> zjs.conf.tmp
-    if [ $BOARD = "arduino_101" ]; then
+    if [ $DEV = "ashell" ]; then
```

Comment:
@brianjjones, I believe these configurations can be used for non-ashell case, but we are not 100% sure why we have to set these for ARC, that's why when I submitted the patch, I didn't want to break the existing configuration.  Maybe when you heard anything back from the Zephyr on the bug you filed, we could adapt these changes for non-ashell.